test/token/ERC721/ERC721.test.js

```diff
@@ -324,7 +324,7 @@ contract('ERC721', function (accounts) {
 
           describe('to a valid receiver contract', function () {
             beforeEach(async function () {
-              this.receiver = await ERC721ReceiverMock.new(RECEIVER_MAGIC_VALUE, false);
+              this.receiver = await ERC721ReceiverMock.new(RECEIVER_MAGIC_VALUE, 0);
```

Comment:
It would be better to define constants for `0`, `1`, ..., so we don't need to consult what they mean.